mp/src/game/server/fortress/tf_basefourwheelvehicle.cpp

```diff
@@ -472,7 +472,8 @@ void CBaseTFFourWheelVehicle::PlayerControlInit( CBasePlayer *pPlayer )
 	pPlayer->SetViewOffset( vec3_origin );
 
 	m_playerOn.FireOutput( pPlayer, this, 0 );
-	InputTurnOn( inputdata_t() );
+	inputdata_t inputdata;
```

Comment:
Passing a temporary via reference is only allowed for `const&`. gcc disallows doing that if the reference is not specified as `const`.
In most cases I simply changed the function parameters to include `const`, however since `InputTurn{On,Off}()` is used in multiple places I was a bit lazy to check if that's a reasonable change and opted to put the temporary in a variable instead.